### DISCUSSION TITLE
How to import transactions?

### DISCUSSION THREAD

**Discussion by mtsz-pl** [ananthakumaran/paisa#137] [Q&A] [answered]
I'm almost sure I'm missing something very simple, but I'm not able to import any transaction. I created a dummy CSV file with one transaction, created a template, and basically nothing happens. Is there some button I should click for transactions to be imported?

Also it would be nice to have some screenshots on https://paisa.fyi/reference/import/ - because I have no idea what is what on the UI

CSV:
```
01-01-2020,Amazon,100,USD
```
Template:
```template
{{#if (isDate ROW.A "DD-MMM-YYYY")}}
  {{date ROW.A "DD-MMM-YYYY"}} {{ROW.B}}
  Expense:Misc      {{ROW.C}} {{ROW.D}}
  Assets:Checking
{{/if}}

```

**Comment by ananthakumaran** [comment, marked as answer]
I am assuming it's related to the date, could you try `"DD-MM-YYYY"`? I am reluctant to add screenshots because they become outdated soon (unlike text) and keeping them updated with the new UI is not easy when things are changing a lot.

**Reply by mtsz-pl** [reply]
🤦 omg you are right about date format, thanks!

Regarding images, I understand.
